lib/host/android_notifications.dart

```diff
@@ -25,3 +25,36 @@ abstract class PendingIntentFlag {
   /// Corresponds to `FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT`.
   static const allowUnsafeImplicitIntent = 1 << 24;
 }
+
+/// For use in [NotificationChannel.importance].
+///
+/// See: https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)
```

Comment:
```suggestion
/// See:
///   https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)
///   https://developer.android.com/reference/android/app/NotificationChannel#getImportance()
```

The getter's doc has some additional information that isn't on the setter, and they don't link to each other, so best to provide both.